### PR TITLE
comments updates

### PR DESCRIPTION
**Pull request by cdolfi**
Updates on in line comments

### CODE REVIEW

File: docs/CONTRIBUTORS.md [hunk 1]
```diff
@@ -3,29 +3,29 @@ If you are interested in making a code contribution and would like to learn more
...
+12. In the description of the pull request, explain the changes that you made, any issues you think exist with the pull request you made, and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you fix any problems and improve it!
+13. Wait for the pull request to be reviewed by a maintainer.
+14. Make changes to the pull request if the reviewing maintainer recommends them.
+15. Celebrate your success after your pull request is merged!
 
 ## Steps to create a new visualization
-1. Make a new file for the visualization and put it in the folder of the [page](https://github.com/sandiego-rh/explorer/tree/main/pages) you want to add it to
+1. Make a copy of the [visualization template](https://github.com/oss-aspen/8Knot/blob/dev/pages/visualization_template/viz_template.py) for the visualization and put it in the folder of the [page](https://github.com/oss-aspen/8Knot/tree/dev/pages) you want to add it to
 2. Once the file is created, import the card to the respective page file and put it into the layout
```
Comment (JamesKunstle) [inline docs/CONTRIBUTORS.md:22]: 'import the card' could be rewritten as 'import the new visualization file'

**Review by JamesKunstle — CHANGES_REQUESTED** [review]
one small change, otherwise lgtm

**Comment by cdolfi** [comment]
@JamesKunstle should be good to go. Only failure is on a "spelling" issue from a link

**Comment by JamesKunstle** [comment]
@cdolfi okay bet. just fix the spelling errors and we'll be good. either fix an actual spelling error ('repective') or add a correctly spelled but unknown word ('oss') to .wordlist.yml in the repo.

**Comment by cdolfi** [comment]
@JamesKunstle Should be good to go now

**Comment by JamesKunstle** [comment]
@cdolfi perfect, thank you!

**Review by JamesKunstle — APPROVED** [review]
@cdolfi lgtm

**Comment by JamesKunstle** [comment]
@cdolfi check rebase error then I'll merge

**Comment by cdolfi** [comment]
@JamesKunstle on my side its showing no conflicts

**Comment by JamesKunstle** [comment]
if you're able to merge it via github that's chill but it's telling me "This branch cannot be rebased due to conflicts" above so idk

**Comment by JamesKunstle** [comment]
huh nice

**Pull request merged**